src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Deadline{
+    private String start;
+
+    public Event(String description, String by, String start) {
+        super(description, by);
+        this.start = start;
+    }
```

Comment:
Do you think Javadoc comments could be good in the other class files as well? They are not hard to understand, but just for convention